ui/lib/core/addon/components/info-table-row.hbs

```diff
@@ -5,9 +5,10 @@
 
 {{#if (or (has-block) this.isVisible)}}
   <div class="info-table-row" data-test-component="info-table-row" ...attributes>
+    {{! label section of the row }}
```

Comment:
The changes in this file are small but **far-reaching**. We had the following selectors:
1. data-test-label-div (the div around a label)
2. data-test-row-label (the label)
3. data-test-value-div (a value in the row wrapped in a non-special div)
4. data-test-row-value (the value if it was the default)

I changed these so we now have the following:
1. data-test-row-label (the label)
2. data-test-row-value (the big one used in lots of places)

The goal of this change was to simplify. You are either searching for the label in the row or the value.